src/api.js

```diff
@@ -49,10 +49,30 @@ const getCommentsbyArticleID = (article_id) => {
     })
 }
 
+const postComment = (article_id, username, body) => {
+    return axios.post(`https://jakes-news-api.herokuapp.com/api/articles/${article_id}/comments`, {
+        username: `${username}`,
+        body: `${body}`
+    }).then((res) => {
+        return res; 
+    })
+}
+
+const getArticleImage = (topic) => {
+    return axios.get(`https://api.unsplash.com/search/photos?query=${topic}&client_id=DjTIHUxEvutYriF2d_T7SrIVNoMNdEruLAWdmZAtzWg`)
```

Comment:
Look up the notes on Axios params [here](https://notes.northcoders.com/courses/js-front-end/react-projects) - this will mean that you can refactor this so you can pass a params object:
<img width="691" alt="Screenshot 2022-08-25 at 09 22 07" src="https://user-images.githubusercontent.com/67956822/186613983-60b312fd-c2ee-43be-9aca-4f929d0f5877.png">

This means that you can just have one 'getArticles' function, which will in turn mean you've got fewer functions in your api.js and employers will be like 'wow this person knows how Axios works' :)

Reply:
Thanks for the comments Sam, I'm still a little confused about how I could just have the one get articles function using the params, does this mean I could use params to get articles and comments on the same api call?